tools/ci_connector_ops/ci_connector_ops/pipelines/publish.py

```diff
@@ -186,7 +186,7 @@ async def _run(self, built_connector: Container) -> StepResult:
         specs_to_uploads: List[Tuple[str, File]] = [(self.oss_spec_key, await self._get_spec_as_file(oss_spec))]
 
         if oss_spec != cloud_spec:
-            specs_to_uploads.append(self.cloud_spec_key, await self._get_spec_as_file(cloud_spec, "cloud_spec_to_cache.json"))
+            specs_to_uploads.append((self.cloud_spec_key, await self._get_spec_as_file(cloud_spec, "cloud_spec_to_cache.json")))
```

Comment:
This was caught by the same Pyright filter because `append` expects only 1 positional argument.